IBPSA/Fluid/Geothermal/Borefields/BaseClasses/Boreholes/BaseClasses/Functions/partialInternalResistances.mo

```diff
@@ -29,6 +29,8 @@ partial function partialInternalResistances
   input Modelica.Units.SI.SpecificHeatCapacity cpMed
     "Specific heat capacity of the fluid";
   input Modelica.Units.SI.MassFlowRate m_flow_nominal "Nominal mass flow rate";
+  input String instanceName="undeclared caller"
+    "Instance name of the model or block that calls this function";
```

Comment:
I know this exact name is used in other models as well, but why the exception to the 3 letter scheme?